doc/source/serve/deployment-graph.md

```diff
@@ -2,13 +2,20 @@
 
 # Deployment Graph
 
-To learn more about deployment graph in an end-to-end walkthrough:
+## Introduction to Deployment Graphs
 
-- [E2E Tutorials](./deployment-graph/deployment-graph-e2e-tutorial.md)
+Ray Serve’s Deployment Graph offers a simple API to compose your Ray Serve deployments together by declaring how to route a request through them. This is particularly useful when using ML model composition or mixing business logic and model inference in your application. Encapsulating each of your models and each of your business logic steps in independent deployments gives you added flexibility to update or scale individual deployments.
```

Comment:
@shrekris-anyscale in general let's only try to expose complexity to users when it's necessary. In this section, I don't think we should introduce the idea of a "graph" at all until we get to the actual DAG API section.

For example, a more incremental flow would be like this:
- This intro says it's going to walk you through how to accomplish model composition (focus on job-to-be-done!), particularly how to compose multiple deployments containing ML models or business logic.
- Then the first section introduces having deployments call into each other (no need to call this a graph at all yet, the user doesn't need to be exposed to the complexity).
- Then the final section introduces the full call graph/DAG API.